packages/preview/polytonoi/0.1.0/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "polytonoi"
+version = "0.1.0"
+entrypoint = "polytonoi.typ"
+authors = ["Dei Layborer"]
+license = "GPL v. 3.0"
```

Comment:
CI failed. This should be `"GPL-3.0-only"` or `"GPL-3.0-or-later"`.

Reply:
Ah ok, thank you.  I saw that it was something related to the license, but my brain decided that it was not up for investigating last night :)